app/javascript/app/components/button/info-button/info-button-component.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from 'components/icon';
+import squareButtonTheme from 'styles/themes/button/button-square';
+import blueIconTheme from 'styles/themes/icon/icon-blue';
+import infoIcon from 'assets/icons/info.svg';
+import ReactTooltip from 'react-tooltip';
+import cx from 'classnames';
+
+import styles from './info-button-styles.scss';
+
+const InfoButton = ({ className, text }) => (
+  <div
+    className={cx(styles.infoButton, className)}
+    theme={squareButtonTheme}
+    data-tip={text}
+    role="tooltip"
+  >
+    <Icon theme={blueIconTheme} icon={infoIcon} />
+    <ReactTooltip />
```

Comment:
It's better if we extract `<ReactTooltip />` component to the page where the buttons are displayed, that way instead of having a tooltip element in the DOM for every `<InfoButton />` we just have one tooltip which content changes depending on the button we are hovering over.  
``` 
<InfoButton text={'hello 1'} />
<InfoButton text={'hello 2'} />
<InfoButton text={'hello 3'} />
<ReactTooltip />
```